sdk/firstload/src/index.tsx

```diff
@@ -112,6 +112,26 @@ const H: HighlightPublicInterface = {
 			}
 			init_called = true
 
+			if (options?.enableOtelTracing) {
+				// Use dynamic import to avoid bundling the opentelemetry dependencies
+				import('@highlight-run/client/src/otel/index').then(
```

Comment:
this is importing the src file, does it get bundled? should we add the otel bundle to the client vite bundling? otherwise, we can probably avoid building client altogether by updating line 137 below to use the src file as well

Reply:
Hmm... Not sure I know what you mean here. Maybe something we can follow up on in pairing tomorrow?

Reply:
i think im realizing that since firstload is built by vite, we can reference the `src` file both here and line 137, rather than having to `vite build` the client and then use the bundled file here. https://www.loom.com/share/15ce6ce813e24f848839c069bebe5234

Reply:
Oh, that's a great point! We should definitely look at simplifying this!

Reply:
https://linear.app/highlight/issue/HIG-4770/eliminate-dependency-on-highlight-runclient